includes/class-banana-crystal-woocommerce.php

```diff
@@ -249,9 +249,11 @@ public function process_payment( $order_id ) {
            $notes .= $product_name.' x '.$quantity.'\n';
         }
                 
-        
+        // Get store's currency
+    	$currency = get_woocommerce_currency();
+
         //redirect user to store banana crystal payment page
-        $params = '?amount='.$order->order_total.'&note='.$notes.'&order_id='.$order_id.'&sd='. base64_encode($order_key);
+        $params = '?amount='.$order->order_total.'&note='.$notes.'&order_id='.$order_id.'&sd='.'&currency='.$currency.'&sd='. base64_encode($order_key);
         $store_user_name = $this->get_option( 'store_username' );
```

Comment:
@Oj-Michael it looks like you have a typo/extra `sd` param in here

```

$params = '?amount='.$order->order_total.'&note='.$notes.'&order_id='.$order_id.'&currency='.$currency.'&sd='. base64_encode($order_key);

``

Reply:
> @Oj-Michael it looks like you have a typo/extra `sd` param in here
> 
> ```
> 
> $params = '?amount='.$order->order_total.'&note='.$notes.'&order_id='.$order_id.'&currency='.$currency.'&sd='. base64_encode($order_key);
> 
> ``
> ```

Yes ma, It is a typo... I will correct it.